src/components/shared/navigationBars/topBar/networkName.js

```diff
@@ -15,10 +17,25 @@ const Network = ({ network, t, token }) => {
   return (
     <section className={`${styles.wrapper} network-status`}>
       <span className={`${styles.status} ${statusColor}`} />
-      <p>
+      <div className={styles.message}>
         <span>{t('Connected to:')}</span>
-        <span>{networksList[activeNetwork]}</span>
-      </p>
+        <Tooltip
+          className={styles.tooltipWrapper}
+          size="maxContent"
+          position="bottom left"
+          content={(
+            <span className="network-name">{networksList[activeNetwork]}</span>
+          )}
+        >
+          <p className="network-address">
+            {
+              network.networks
+                ? network.networks[token]?.serviceUrl
```

Comment:
@reyraa The network URL here is capitalized via the CSS and displays as 

<img width="1245" alt="Network tooltip" src="https://dl.dropbox.com/s/kd0oi79skpfy7sy/shot_210930_154737.png" />